.github/workflows/build.yaml

```diff
@@ -24,3 +24,19 @@ jobs:
       - uses: golangci/golangci-lint-action@v3
         with:
           version: v1.52.2
+  # lint-bundle is here to dogfood the setup-regal action from https://github.com/StyraInc/setup-regal
+  lint-bundle:
+    name: Lint Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

Comment:
Might as well make this v4 right away, as it's used elsewhere in this file

Reply:
👍 fixed